### PR TITLE
Mobile version with form section finish

### PR DESCRIPTION
**Pull request by SanaTameem**
### In this section:
- [ ]  I made the contact form of portfolio project:
- [ ] All required client-side validations have been added.
- [ ] To lay out the elements on the page I have used Flexbox.
- [ ] And I have Implemented the button interactions (hover).

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @SanaTameem, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project, that doesn't have a pull request description, kindly add one. See a demo on how to add a PR description below:

pull_request_desc
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -211,10 +212,36 @@ <h4 class="languages-heading" >Languages</h4>
...
         </div>
       </section>
 
+      <!--Form section-->
+      <section class="form-container">
+      <form action="https://formspree.io/html" method="post" class="my-form">
+        <ul class="form-list">
+          <li class="form-list-items"><input type="text" name="userName" id="name-input" placeholder="Full name" maxlength="30" required></li>
+          <li class="form-list-items"><input type="email" name="userEmail" id="email-input" placeholder="Email address" required></li>
+          <li class="form-list-items"><textarea name="userMessage" id="message" cols="30" rows="10" maxlength="500" required>Enter text here</textarea></li>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please from the project requirement, you should make sure that all fields have placeholder text but your `textarea` field does not have any placeholder text. So kindly add a placeholder text to the `textarea` field :pray: 
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @SanaTameem , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @SanaTameem,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Form tag ✔️ 
- input attributes ✔️ 
- correct use of formspree ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**